internal/services/signalr/signalr_service_resource.go

```diff
@@ -73,11 +73,26 @@ func resourceArmSignalRServiceCreate(d *pluginsdk.ResourceData, meta interface{}
 	}
 
 	sku := d.Get("sku").([]interface{})
-	featureFlags := d.Get("features").(*pluginsdk.Set).List()
-	connectivityLogsEnabled := d.Get("connectivity_logs_enabled").(bool)
-	messagingLogsEnabled := d.Get("messaging_logs_enabled").(bool)
-	liveTraceEnabled := d.Get("live_trace_enabled").(bool)
-	serviceMode := d.Get("service_mode").(string)
+	var featureFlags []interface{}
+	if d.Get("features") != nil {
+		featureFlags = d.Get("features").(*pluginsdk.Set).List()
+	}
+	connectivityLogsEnabled := false
+	if d.Get("connectivity_logs_enabled") != nil {
```

Comment:
we could change connectivityLogsEnabled, messagingLogsEnabled and serviceMode to be more like this:

```suggestion
	if d.Get("connectivity_logs_enabled") {
	  connectivityLogsEnabled = true
```

Reply:
sorry I don't get it, it looks like ResourceData's Get function's signature is:

```golang
func (d *ResourceData) Get(key string) interface{} {
	v, _ := d.GetOk(key)
	return v
}
```

can we use an `interface{}` as a guard expression?